src/ITensorNetworks.jl

```diff
@@ -46,6 +46,7 @@ using ITensors:
   dim,
   orthocenter,
   ProjMPS,
+  add,
```

Comment:
To import `add` to overload it, you need to use `import ITensors: add`. `using ITensors: add` just brings it into scope but doesn't make it overloadable.

Reply:
I see, changed that now